test/e2e/common.go

```diff
@@ -61,13 +62,31 @@ func Byf(format string, a ...interface{}) {
 	By(fmt.Sprintf(format, a...))
 }
 
-func setupSpecNamespace(ctx context.Context, specName string, clusterProxy framework.ClusterProxy, artifactFolder string) (*corev1.Namespace, context.CancelFunc) {
-	Byf("Creating a namespace for hosting the %q test spec", specName)
+func setupSpecNamespace(ctx context.Context, namespaceName string, clusterProxy framework.ClusterProxy, artifactFolder string) (*corev1.Namespace, context.CancelFunc) {
+	Byf("Creating a namespace for hosting the %q test spec", namespaceName)
```

Comment:
`namespaceName` is not the test spec name so maybe we can add `specName` as part of the function signature

Reply:
The speName bring much here, how about rewording to `Byf("Creating namespace %q for hosting the cluster", namespaceName)`